docs/contributing/04-prow-workflow.md

```diff
@@ -0,0 +1,91 @@
+
+If you see any unusual behaviour in Prow, [raise an Issue](https://github.com/kyma-project/test-infra/issues/new) in the test-infra repository, or contact kyma-project/test-infra maintainers.
+
+## OWNERS
+
+The OWNERS file is the Prow feature that allows you to create an ownership structure across the entire repository.
+This file consists of the list of `approvers`, `reviewers`, and `labels`.
+
+The basic structure of this file is as follows:
+```yaml
+reviewers:
+  - githubUser1
+  - githubUser2
+  - githubUserN 
+approvers:
+  - githubUser1
+  - githubUser2
+  - githubUserN
+labels: # optional   
+  - label1
+  - label2
+```
+
+Having the same set of reviewers and approvers is considered a bad practice and should be avoided!
+Reviewers look for general code quality, correctness, sane software engineering, style, etc.
+Approvers look for holistic acceptance criteria, including dependencies with other features, forwards/backwards compatibility, API and flag definitions, etc.
+When a person is in both reviewers and approvers groups, their Approving review is also considered as an approve and marks the PR as `approved`. That behaviour skips the requirement for a second review and immediately allows the PR to be merged.
+
+You can use multiple OWNERS files across the entire repository to diversify the ownership of specific directories between multiple people.
+
+For more information, refer to the [Kubernetes documentation](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md) regarding OWNERS file as well as the [ANNOUNCEMENTS.md](https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md) file on the kubernetes/test-infra repository.
+
+## OWNERS_ALIASES
+
+The OWNERS_ALIASES file contains a list of aliases that can be used in the OWNERS files, instead of the GitHub usernames.
+It is useful when the same set of people is responsible for bigger amount of directories in the repo.
+
+The example structure of this file is as follows:
+```yaml
+aliases:
+  owners-group:
+    - githubUser1
+    - githubUser2
+    - githubUserN
+```
+
+For more information, refer to the official [Kubernetes documentation](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners_aliases).
+
+## Code review process
+
+Since in Kyma Prow is responsible for ensuring the review process, our flow is the same as in the official [Kubernetes documentation](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process).
+
+## Commands
+
+Prow provides GitHub automations and ChatOps using `/foo` styled slash-commands. You can use these command by creating a comment under the GitHub Issue or Pull Request.
+This flow can be extended by various built-in plugins and can be further extended with external integrations with Prow API that it provides. 
```

Comment:
```suggestion

```

Reply:
I know that you know a lot and that you want to share knowledge, but the reader doesn't need this information here 🙂